examples/boto3/README.md

```diff
@@ -14,11 +14,20 @@ The standard [AWS CLI](https://docs.aws.amazon.com/cli/latest/) may also be used
 aws --endpoint-url http://localhost:8000 s3api list-objects --bucket=mybucket --allow-unordered
 ```
 - Bucket notifications with filtering extensions:
+  Note that sns = signature_version = s3
```

Comment:
Please change to:

Note that the following lines should be added to `.aws/credentials`:
```
sns =
    signature_version = s3
```